ac/ac-ck-board/src/ActivityRunner.js

```diff
@@ -26,27 +27,30 @@ const BoardPure = ({
     dataFn.objInsert((ui.y + offsetHeight) * scaleY, [i, 'y']);
   };
 
-  const List = data.map((y, i) => (
-    <div key={y.id}>
-      <ObservationContainer
-        setXY={(_, ui) => setXY(i, ui)}
-        openInfoFn={() => setInfo(y)}
-        title={y.title}
-        scaleY={scaleY}
-        scaleX={scaleX}
-        content={y.content}
-        x={y.x / scaleX - offsetWidth}
-        y={y.y / scaleY - offsetHeight}
-      />
-    </div>
-  ));
   if (!width || !height) {
     return null;
   }
   return (
-    <div style={{ height: '100%', width: '100%' }}>
-      {config.quadrants && (
-        <Quadrants config={config} width={width} height={height} />
+    <div
+      style={{
+        width: width + ' px',
+        height: height + ' px',
+        position: 'relative'
+      }}
+    >
+      {config.allowCreate && (
+        <div
+          style={{ zIndex: 9, position: 'absolute', right: '0px', top: '0px' }}
```

Comment:
Prastut would be saddened by this amount of inline style, but I'm fine with it given the mess that this activity used to be.